### ISSUE TITLE
Large value of RMSE while fine-tuning 7net-0

### ISSUE THREAD

**Issue by DGKang234** [MDIL-SNU/SevenNet#250]
Dear Developers,

I attempt to fine-tuning 7net-0 model with the vasp results of the single point calculated randomly scattered atoms.
During the fine-tuning (I followed the fine-tuning tutorial) the RMSE of each epoch is very high:

```
Epoch 10/10  Learning rate: 0.000400
------------------------------------------------------------------------------------------------------------------------
       Energy_RMSE (eV/atom)   Force_RMSE (eV/Å)   Stress_RMSE (kbar)   Energy_MAE (eV/atom)   Force_MAE (eV/Å)   Stress_MAE (kbar)   Energy_Loss   Force_Loss   Stress_Loss   TotalLoss  
------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
Train             178.687156           81.152464         68301.612801              82.339354          16.575042         8404.350185      0.823344     0.165701     84.043455    1.829479
Valid             183.634165           91.979268         76253.958309              92.050893          19.477570        10124.414716      0.920459     0.194726    101.244100    2.127626
Epoch 10 elapsed: 0:00:23.23
```

Could you please give advice to point out what I am doing wrong?

P.S.

I used INCAR for the single point;
```
PREC = Accurate       # Precision of the calculation
ENCUT = 520           # Plane-wave cutoff energy

# Electronic Relaxation
EDIFF = 1E-6 #5          # Energy convergence criterion for electronic self-consistency
ALGO = Normal           # Algorithm for electronic minimisation

# Ionic Relaxation
IBRION = 2            # Conjugate gradient algorithm for ionic relaxation
NSW = 0 #1000             # Maximum number of ionic steps
EDIFFG = -0.03        # Force convergence criterion (negative sign indicates force)
ISIF = 2              # Relaxation of atomic positions, calculation of stress tensor

# Output Settings
LCHARG = .FALSE.      # Do not write charge density
LWAVE = .FALSE.       # Do not write wavefunctions
```

Cheers,

Dr. Kang

**Comment by DGKang234** [comment]
Hi Dr Park,
Thank you for the reply.
1. I have not tried to use .yaml for training. I will try fintuning with .yaml format: should I just edit the path of the trianing data? (apologise for the newbie question)

2. If I train with MPtrj and my training data (scattered atoms/liquid of atoms) should I train from scratch or finetuning would be okay?

3. yes, I will use the recent model to train/finetune.

Thank you.
Dr. Kang

**Comment by DGKang234** [comment]
I love to use the reEWC (https://github.com/kskjs1203/SevenNet-reEWC) for the finetuning.
I wonder when would it be available in this repository?

in the mean time, I want to work it out to make the potential to describe the designated systems with fair accuracy which I will follow your suggestion; replay method: omat potential <-- finetune w/ omat data + finetuning data.
